test/unit/effectsTest.js

```diff
@@ -201,3 +203,70 @@ test('logs invalid background palette', async t => {
   nativeAPI.reset();
   sinon.restore();
 });
+
+
+test('Set background effect to none, unknown, and known effects', async t => {
```

Comment:
This test was copied over from `effects.js` (which was deleted, as it seems redundant with this file)